app_dart/lib/src/service/scheduler.dart

```diff
@@ -117,7 +117,7 @@ class Scheduler {
   }
 
   Future<void> _addCommit(Commit commit) async {
-    if (!Config.schedulerSupportedRepos.contains(commit.slug)) {
+    if (!Config.supportedRepos.contains(commit.slug)) {
```

Comment:
This is the key logic. Moving to supportedRepos will support flutter, engine, plugins, packages, cocoon

Reply:
This will write all tasks to datastore as well. How about the task status update? I guess we will wait for the result processing as the current refresh API seems already overloaded just considering framework repo.

Reply:
update-task-status is hard coded to only work for the framework - https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/request_handlers/update_task_status.dart;l=111;drc=fddb3f5f9887e9621b1650bf36a345398847f724

The plan is to deprecate it in favor of the results processing microservice

Reply:
For the results being overloaded, I plan to add cron jobs for each separate repo (which shouldn't be an issue as flutter/flutter has by far the most tasks). Early Q1 that will no longer be necessary as I'll have everything wired to update via pubsub